crates/api/src/chronicle_graphql/mod.rs

```diff
@@ -741,6 +764,13 @@ where
             }
         };
 
+        if sec.open {
+            tokio::spawn(async move {
+                tokio::time::sleep(Duration::from_millis(200)).await;
+                open::that(format!("http://{}", address)).ok();
```

Comment:
Are we okay to throw away any `Err` here?

Reply:
As I say, this is just replacing what we once had so that things work, for now.